feat: prepare for new lz

marnixdessing opened the pull request:
- Added configuration object
- Added configuration for existing and new lz branches

marnixdessing committed:
feat: prepare main file for new lz

marnixdessing committed:
feat: add permissions boundary aspect

marnixdessing committed:
feat: set prod account id correctly

marnixdessing committed:
fix: wrong account number

marnixdessing committed:
feat: make sure cloudfront does not yet listne to the *.nijmegen.nl domain name (failing deployment)

marnixdessing committed:
fix: added permission boundary to pipeline stack

marnixdessing committed:
feat: disable privileged mode on pipeline

marnixdessing committed:
feat: do not deploy a DS record if no value is set

marnixdessing committed:
feat: added correct branch configurations

marnixdessing committed:
chore: eslint

joostvanderborg committed:
fix: Insights layer arn was not region independent. (#553)

joostvanderborg committed:
fix: Update cloudfront logs bucket (#554)

Fixes issue
```
Resource handler returned message: "Invalid request provided:
AWS::CloudFront::Distribution: The S3 bucket that you specified for
CloudFront logs does not enable ACL access:
mijn-api-cloudfront-stack-cloudfrontlogs```

marnixdessing committed:
fix: keep out the nijmegen.nl domain in new-lz (temp)

Commit:
chore: self mutation

Signed-off-by: github-actions <github-actions@github.com>

marnixdessing committed:
feat: use a config object in the new lz setup

marnixdessing committed:
fix: switch default branch 'development' -> 'acceptance-new-lz'

by preparing for the new lz old branches are removed from this version

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/5733187190

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #571 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
Merge branch 'feat/config-object-in-new-lz' into feat/update-accp-with-new-lz-stuff

marnixdessing committed:
feat: implement configuration and lz upgrade on existing environments

marnixdessing committed:
feat: added existing ds record values

marnixdessing committed:
fix: pipeline stack cdk name

marnixdessing committed:
fix: failing test due to new config prop

marnixdessing committed:
Merge pull request #573 from GemeenteNijmegen/feat/update-accp-with-new-lz-stuff

feat: update accp (in new lz) with new lz stuff and new changes on accp

marnixdessing committed:
fix: add proper pipeline name before replacing

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/5745103641

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #575 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
Merge branch 'acceptance' into feat/update-accp-with-new-lz-stuff

marnixdessing committed:
Merge pull request #577 from GemeenteNijmegen/acceptance-new-lz

fix: keep this branch up to dat with new lz changes

marnixdessing committed:
Merge pull request #572 from GemeenteNijmegen/feat/update-accp-with-new-lz-stuff

feat: update accp with new lz stuff

joostvanderborg reviewed (approved):
Gecheckt in accp, werkt ok

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/mijn-nijmegen/actions/runs/5756937447

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #579 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
chore: trigger build

marnixdessing committed:
Merge pull request #580 from GemeenteNijmegen/chore/trigger-build-1

chore: trigger build
